ui/src/components/home/HomeContainer.js

```diff
@@ -38,9 +39,10 @@ const HomeContainer = (props) => {
   };
   return (
     <Flex direction="column">
-      <HeroSection />
-      <EventsSection />
+      <HeroSection sermonRef={sermonRef} />
+      <HelloSermonSection ref={sermonRef} />
```

Comment:
can you check your work using iphoneSE responsive mode? there's a lot of small but obvious misalignments. e.g.:
![Screenshot 2022-08-19 at 17 03 55](https://user-images.githubusercontent.com/33679483/185584899-7c31483a-c150-4c1d-9c52-a315ccf707aa.png)
![Screenshot 2022-08-19 at 17 05 00](https://user-images.githubusercontent.com/33679483/185585106-1d078b88-06ef-453b-848f-0b235a8504bc.png)